proyecto 1 BODAS/style.css

```diff
@@ -15,29 +18,64 @@ body {
 
 .home h2 {
   display: inline-block;
-  margin: 0px 120px;
+  margin: 10px 120px;
   box-sizing: border-box;
   background-color: #a0e8af;
   font-family: "Imperial Script", cursive;
   text-align: center;
-}
-
-.home .bendicion {
   color: #86baa1;
   width: 500px;
   border-style: solid;
   border-color: aliceblue;
 }
 
-.home .boda {
-  color: #86baa1;
+.home img {
+  /* margin: 10px 120px; */
   width: 500px;
-  border-style: solid;
-  border-color: aliceblue;
 }
 
-.home img {
-  margin: 10px 120px;
+.home {
+  position: relative;
+}
+
+.home .evento1-1,
+.evento1 {
+  position: absolute;
+  top: 270px;
+  left: 120px;
+}
+
+.home .evento1-1 {
+  z-index: -1;
+}
+
+.home .evento1 {
+  transition-property: all;
```

Comment:
this will use a lot of memory, is not efficient, is better to just use transition on the property you need, for example just margin or background color, please take a look on this link: 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x